### PR TITLE
TUNE-1003 choose clinic template

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@moh-moola @saeedmb8 

Please review:
screen shot 2018-09-26 at 16 24 03

File: tuneme/static/new/js/main.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+"use strict";
```
Comment (moh-moola) [inline tuneme/static/new/js/main.js:1]: Is the JS necessary though?

Comment (Mitso) [reply tuneme/static/new/js/main.js:1]: @moh-moola we need to get the form values and populate them as they are being selected. Might not need JS once form is solved

Comment (moh-moola) [reply tuneme/static/new/js/main.js:1]: Doesn't the `selected` attribute of the option element take care of that, we can add it dynamically for a prepopulated form

Comment (moh-moola) [reply tuneme/static/new/js/main.js:1]: oh I get it now ...for the label, but will we just disregard this feature for feature phones or maybe we need to rethink this field's label?

**Review by moh-moola — COMMENTED** [review]
I have QA'd the radius filter selection issue which the JS was trying to solve and it is not an issue on the tip of the develop branch. In my opinion I think we can do away with the JS

**Review by moh-moola — APPROVED** [review]

**Pull request merged**